Sources/BadgeGenerator/BadgeGenerator.swift

```diff
@@ -110,3 +110,96 @@ extension UIView {
     }
     
 }
+
+#if DEBUG
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+struct BadgeView: UIViewRepresentable {
+    
+    private let view = UIView()
```

Comment:
I try to avoid using names that `self` is using. In this case since you can't change the parameter name, it would be good to change the class property. Swift is great with naming collisions, but I do this in order to avoid the possibility of confusing properties. I do believe it may require some extra compile time as well, but I'm not sure about that
```suggestion
    private let previewView = UIView()
```